dockerfiles/deb.dockerfile

```diff
@@ -43,6 +43,7 @@ WORKDIR /root/containerd
 # NOTE: not using a cache-mount for apt, to prevent issues when building multiple
 #       distros on the same machine / build-cache
 RUN apt-get update -q && apt-get install -y --no-install-recommends \
+    ca-certificates \
```

Comment:
I guess alternatively, we could call [`update-ca-certificates`](https://manpages.ubuntu.com/manpages/xenial/man8/update-ca-certificates.8.html) not sure what's better, I guess both work